src/components/VPopover/VPopoverContent.vue

```diff
@@ -1,15 +1,20 @@
 <template>
   <div
     v-if="visible"
-    ref="popoverRef"
-    :class="[$style.popover, { [`z-[${zIndex}]`]: zIndex !== undefined }]"
+    class="w-0"
     :aria-hidden="!visible"
-    :tabindex="typeof $props.tabindex !== 'undefined' ? $props.tabindex : -1"
     v-on="$listeners"
     @keydown="onKeyDown"
-    @blur="onBlur"
   >
-    <slot />
+    <div
+      ref="popoverRef"
+      class="bg-white border border-light-gray rounded-sm shadow max-w-max"
+      :class="{ [`z-[${zIndex}]`]: zIndex !== undefined }"
+      :tabindex="typeof $props.tabindex !== 'undefined' ? $props.tabindex : -1"
```

Comment:
```suggestion
      :tabindex="typeof tabindex !== 'undefined' ? tabindex : -1"
```

Reply:
Looking at this now, it must have meant to have been `$attrs` instead of `$props` because there's no `tabindex` prop... but I cannot figure out why this was necessary. We don't pass tabindex in at VPopover or anywhere else VPopoverContent is used so it seems like dead code to me. VPopover doesn't accept it as a prop that it forwards either so it's not even technically possible for any consumer of VPopover to pass it.

Anyway, set it to constant of -1 in https://github.com/WordPress/openverse-frontend/pull/870/commits/43d0c29d3a541b364286db22f79d868f4e0d30e9